uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -763,19 +763,14 @@ sub in_range
     return 0 unless $range_string;
     return 0 unless defined($value);
 
-    my @ranges = split(/,/, $range_string);
-
-    my $range = 0;
-    foreach $range (@ranges) {
-       chomp($range);
-       if($range=~/^[0-9.]+$/) { ## single value element
-           return 1 if &floats_are_equal($value, $range, $FLOAT_EQUALS_NB_DECIMALS);
-       } else { ## range X-Y
-           $range =~ /([0-9.]+)-([0-9.]+)/;
-           return 1 if ($1 <= $value && $value <= $2) 
-               || &floats_are_equal($value, $1, $FLOAT_EQUALS_NB_DECIMALS) 
-               || &floats_are_equal($value, $2, $FLOAT_EQUALS_NB_DECIMALS);
-       }
+    chomp($range_string);
+    if($range_string=~/^[0-9.]+$/) { ## single value element
+        return 1 if &floats_are_equal($value, $range, $FLOAT_EQUALS_NB_DECIMALS);
```

Comment:
in the function `&floats_are_equal`, `$range` should be replaced by `$range_string`